app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,107 +1,53 @@
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
-import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
 import EmptyState from "components/Common/EmptyState";
 
-import DeleteAlert from "./DeleteAlert";
-import NewNotePane from "./Pane/Create";
-import Table from "./Table";
+import { NOTES_CARD_DATA as notes_content } from "./constants";
```

Comment:
You are not going through the React Best Practices very well. I want you to be extra careful and go through the materials very well.

Read: https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react#:~:text=Use%20upper%20snake%20case%20in%20constant%20name